src/config.ts

```
@@ -146,37 +140,30 @@ export async function createViemClient(
 /**
  * Keeps record of http fetch requests for a http viem client
  */
-export function onFetchRequest(request: Request, rpcRecords: Record<string, RpcRecord>) {
+export function onFetchRequest(request: Request, rpcState: RpcState) {
     let url = request.url;
     if (!request.url.endsWith("/")) url = url + "/";
-    let record = rpcRecords[url];
+    let record = rpcState.metrics[url];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)